src/index.js

```diff
@@ -17,7 +17,7 @@ export default (schema, data, errors, options = {}) => {
   });
 
   if (format === 'cli') {
-    return customErrors.map(customErrorToText).join();
+    return customErrors.map(customErrorToText).join('\n\n');
```

Comment:
Without this change, the error title starts on the same line as the previous error’s code block.

This makes output start on a new line.